### DISCUSSION TITLE
"File exists (add ! to override)" on all debug information is limited how can I get full verbosity of this plugin and find fundamental issue?

### DISCUSSION THREAD

**Discussion by jpmorris** [vscode-neovim/vscode-neovim#2189] [Q&A]
How can I debug this issue:  I've enabled `--clean`. I disabled all plugins but neovim. I've tried neovim 10.0 and now 9.5.  9.5 was working for a time when I completely erased my profile and started over.  But the problem is back. The directory is in a Syncthing related  directory running off an NFS unraid drive.   I believe this MAY have to do with something unraid (or maybe syncthing) related (extended file information ) because when I save outside this directory it seems to work.

Google says that this error may be related to nvim-treesitter (when it has issues opening hte file). I SHOULD have nvim-tresitter disabled when it's in vscode in the lua config (not sure if the disable is working).

Basically I would just like to know how to approach this and debug it (whether or not it's unraid related as trying to find issues with unraid/nfs direclty has not been fruitful).   Iv'e enabled debugging though the debug console  and it only shows the same `File exists (add ! to override)`  error, and the stack is so filled with calls not essential to this issue it's not helping me.  I can probably fix it by blanking out my profile again but that doesn't help me find the root problem which seems to reoccur.  'vscode-messages' output shows nothing. 

Is there any way to increase the verbosity (in console or `vscode-neovim messages` output) or some sort of debugging method to try and narrow down the fundamental error that is causing this?

**Comment by jpmorris** [comment]
- deleted all plugins from neovim (only remaps and a couple of files)
- blanked out my settings.json and keybindings.json
- turned off my syncthing
- deleted my ~/.config/Code directory
- renamed my extensions folder and only installed vscode-neovim

**Comment by jpmorris** [comment]
This problem starts with vscode-neovim >= 1.18.0 I'm going to submit a bug

**Comment by xiyaowong** [comment]
@jpmorris Is this issue occurring when you execute `:w`?

**Comment by jpmorris** [comment]
yes it is.  

Sorry I no longer have the issue as I played around with it (deleting and recreating profiles), but I STILL couldn't find out what is causing the problem.

**Comment by cs-clarence** [comment]
having this issue as well

**Reply by jpmorris** [reply]
Sorry I dont have a good fix for you.  Try older versions of neovim (I think that worked for me). You can also try newer versions, Try resetting everything, disable all plugins, reinstall, etc.  If you're running in nfs/cifs then try running off that drive. And post back here if you find any solutions.

**Reply by cs-clarence** [reply]
in my case, it worked by rolling back to the 1.17.x version of the plugin, no need to rollback the neovim version as well

**Reply by ahelwer** [reply]
I can confirm that after wasting a bunch of time messing around with my vim config, just reverting to version 1.17.x of the extension fixed the issue for me: https://stackoverflow.com/a/53755378/2852699

**Reply by jpmorris** [reply]
Got this error again, thank you both for your researched, saved me some time.

**Reply by cs-clarence** [reply]
i've taken a look at the code and there were a lot of changes regarding the way that the buffer is saved between 1.17  and 1.18. I'd try to understand and fix but I don't have much free time so it's best to convert this discussion into an issue so that the maintainer is aware.

**Reply by ahelwer** [reply]
I did open an issue https://github.com/vscode-neovim/vscode-neovim/issues/2284

**Reply by ahelwer** [reply]
@cs-clarence and @jpmorris we figured out the issue for me, it's because I was operating in a symlinked directory.

**Reply by cs-clarence** [reply]
@ahelwer I'm using a symlinked directory as well

**Reply by jpmorris** [reply]
Got this problem again and looked it up and, sure enough, a solution was found. Thanks! Symbolic Link here too.

**Comment by justinmk** [comment]
To get debug logs , see https://github.com/vscode-neovim/vscode-neovim?tab=readme-ov-file#troubleshooting . And click the gear icon, select "Debug", then click it again and select "Set as default".
